### ISSUE TITLE
Twitter changed UI

### ISSUE THREAD

**Issue by nnn1590** [Bl4Cc4t/GoodTwitter2#325]
**Please check the following before you continue**
- [x] Checked if the feature has not been requested before

**Describe the feature**
Recently, [Twitter has changed its UI](https://twitter.com/TwitterDesign/status/1425505308563099650).
Some borders have been removed
and the color of the follow button, etc. has changed.

![Trends](https://user-images.githubusercontent.com/28985763/129078847-a05b4b2f-3fe5-4dac-aeb2-9e15f29c763b.png)
![Screenshot of https://twitter.com/search?q=Twitter&f=user&lang=en](https://user-images.githubusercontent.com/28985763/129077738-8728c61a-b0c7-414f-9c70-71aaff2ec544.png)
![@Twitter's twitter profile](https://user-images.githubusercontent.com/28985763/129082513-9ee87f93-91d8-474d-b865-8a8bf2082e73.png)
(and seems Square Avatars broken.)

I would like them to be restored.

**Comment by nanakisan** [comment]
Can confirm the situation
![image](https://user-images.githubusercontent.com/2136875/129088165-12a40d04-cbc8-4ae0-901c-cef8b5ae25df.png)

**Comment by Aridow** [comment]
They explained the changes here : https://twitter.com/TwitterDesign/status/1425505308563099650
The font seems thinner and weird also.
I'l like these changes to be reverted too.

**Comment by MistressAshai** [comment]
Yeah, it's the issue from a while back that was only pushed out to certain unlucky users, now it seems everyone has it.
#266 #272

**Comment by Trazel-Apeally** [comment]
If anyone wants a temporary fix since Bl4Cc4t's on Hiatus until September, I've been (albeit somewhat half-assedly) patching the cracks with custom CSS script
I haven't alterred the pages for accounts already being followed
![Following](https://user-images.githubusercontent.com/15951036/129180597-a5c79921-4cab-4088-b0eb-ccfdc4886071.png)

But I have (hopefully) fixed it for unfollowed accounts:
![Unfollowed](https://user-images.githubusercontent.com/15951036/129181010-bc40a3fb-47b5-4a7f-b91b-7d88b4bbd00f.png)

**Comment by Bl4Cc4t** [comment]
I just pushed the new version, I hope it fixes the majority of changes the UI update introduced.
See you in about a month!

**Comment by Aridow** [comment]
Thanks!
I also noticed when there's a picture in a tweet, the space between the picture and the text above is smaller than before: 
![image](https://user-images.githubusercontent.com/65539338/129279895-d097e059-ced6-46f2-90d1-f7edf83c4c0a.png)
In some cases, there's 0 pixel between them. This seems caused by the update.

**Comment by Trazel-Apeally** [comment]
> Thanks!
> I also noticed when there's a picture in a tweet, the space between the picture and the text above is smaller than before:
> ![image](https://user-images.githubusercontent.com/65539338/129279895-d097e059-ced6-46f2-90d1-f7edf83c4c0a.png)
> In some cases, there's 0 pixel between them. This seems caused by the update.

If you're fine with using CSS to fix it for now, this is what I've been using to patch it for now:
`
[class="css-1dbjc4n r-1iusvr4 r-16y2uox r-1777fci r-1t982j2"]>.css-1dbjc4n>.css-1dbjc4n>[class*="css-901oao"]{
padding-bottom: 5px;
}`

**Comment by Bl4Cc4t** [comment]
> 
> 
> Thanks!
> I also noticed when there's a picture in a tweet, the space between the picture and the text above is smaller than before:
> ![image](https://user-images.githubusercontent.com/65539338/129279895-d097e059-ced6-46f2-90d1-f7edf83c4c0a.png)
> In some cases, there's 0 pixel between them. This seems caused by the update.

You're right, in this case they seem to have removed some spacing. I'm not sure why they did that, it looks horrible like that.
I fixed that in https://github.com/Bl4Cc4t/GoodTwitter2/commit/df0262606b0de7e1b321508e7ff7eed82c48dafc.
